test/IntegrationTests/Helpers/MockMetricsCollector.cs

```diff
@@ -292,9 +277,36 @@ private void HandleHttpRequests(HttpListenerContext ctx)
             var metricsMessage = ExportMetricsServiceRequest.Parser.ParseFrom(ctx.Request.InputStream);
             if (metricsMessage.ResourceMetrics != null)
             {
-                foreach (var metrics in metricsMessage.ResourceMetrics)
+                foreach (var rMetrics in metricsMessage.ResourceMetrics)
```

Comment:
suggestion: `resourceMetrics`- I do like the short names and abbreviations but the style on the repo (and C# in general) is longer names.

Reply:
anyway if changing here needs to change `sMetrics` below

Reply:
Addressed 190cf53